ci.yml

```diff
@@ -0,0 +1,162 @@
+# These resource types are here until the PRs get merged in upstream.  :)
```

Comment:
Could you cc me on these upsteam PR? I am curious to look at them. Thanks :)

Reply:
Here they are.

https://github.com/concourse/git-resource/pull/172
https://github.com/jtarchie/github-pullrequest-resource/pull/140
https://github.com/jtarchie/github-pullrequest-resource/pull/138